website/app/components/doc/page/sidebar.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+
+export default class DocPageSidebarComponent extends Component {
+  get navigationTree() {
```

Comment:
it might be useful to add some comments as to what this function is doing so we can better understand it

Reply:
OK